packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -595,6 +598,48 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupSlasPrivateClientProxy(app, options) {
+        if (options.useSlasPrivateClient) {
+            const clientSecret = process?.env?.SLAS_PRIVATE_CLIENT_SECRET
+            if (!clientSecret) {
+                app.use('/ssr/auth', (_, res) => {
+                    return res.status(501).json({
+                        message:
+                            'Environment variable SLAS_PRIVATE_CLIENT_ID not set: LINK_TO_DOC HERE'
```

Comment:
TODO - update this with a link to the docs that explain how to set up a SLAS private client